src/py/flwr/server/strategy/dp_strategy_wrapper_fixed_clipping.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Central differential privacy with fixed clipping.
+
+Papers: https://arxiv.org/pdf/1712.07557.pdf, https://arxiv.org/pdf/1710.06963.pdf
+"""
+from typing import Dict, List, Optional, Tuple, Union
+
+from flwr.common import (
+    EvaluateIns,
+    EvaluateRes,
+    FitIns,
+    FitRes,
+    NDArrays,
+    Parameters,
+    Scalar,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.common.differential_privacy import (
+    add_gaussian_to_params,
+    compute_clip_model_update,
+)
+from flwr.server.client_manager import ClientManager
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy.strategy import Strategy
+
+
+class DPStrategyWrapperServerSideFixedClipping(Strategy):
+    """Wrapper for Configuring a Strategy for Central DP with Server Side Fixed
+    Clipping.
+
+    Parameters
+    ----------
+    strategy: Strategy
+        The strategy to which DP functionalities will be added by this wrapper.
+    noise_multiplier: float
+        The noise multiplier for the Gaussian mechanism for model updates.
+        A value of 1.0 or higher is recommended for strong privacy.
+    clipping_norm: float
+        The value of the clipping norm.
+    num_sampled_clients: int
```

Comment:
Do we need this value to be passed upon initalisation? can we instead compute it in `aggregate_fit()`?

Reply:
Actually, I considered that, but since the amount of noise is computed with respect to the number of clients sampled, I thought it would be a good way to draw the user's attention to the fact that the value is important and related to differential privacy. This is also what TFF does. 

However, on the other hand, removing it would be better for a cleaner api. 
I will remove it for now. Do you also suggest that?

Reply:
It depends if the DP method here implemented accounts for clients dropping. Should the amount of noise added be a function of the number of clients that get back to the server or the number of clients that were sampled?

Reply:
The number of clients that were sampled.

Reply:
It seems like it is not possible to infer the number of sampled clients and only the number of clients that returned the params. Right?